coast/__init__.py

```diff
@@ -0,0 +1,4 @@
+from COAsT.COAsT import COAsT
```

Comment:
We'll need to refactor object names to match Python guidelines at some point, but not crucial at this stage.